src/Merchants/Resources/views/storefront/component/merchant/banner.html.twig

```diff
@@ -8,7 +8,7 @@
         </div>
 
         <div class="button-container merchant-float">
-            <a class="btn btn-merchant" href="https://downtowns.io/">Als Händler registrieren</a>
+            <a class="btn btn-merchant" href="https://downtowns.io/" target="_blank">Als Händler registrieren</a>
```

Comment:
@PaddyS Ich habe hier mal ein target blank hinzu gefügt wenn du das doof findest bitte ich um ein veto. ;)

Reply:
Nope, that's fine!
But we definitely need to fix the URL once the "new domain" is available. :-P

Reply:
Jep thats true. If you wish we could leave this pull request open until we know the new URL ? :thinking:

Reply:
We have a env variable `MERCHANT_PORTAL`. Maybe we should use that as link

Reply:
Done :+1: